src/components/Dialog.tsx

```diff
@@ -243,17 +249,12 @@ const getAnimation = (animationType?: DialogAnimationType) => {
   }
 }
 
-const AnimationWrapper = styled.div<{ animationType?: DialogAnimationType }>`
-  ${Modal} {
-    ${({ animationType }) => getAnimation(animationType)}
-  }
-`
-
-const FullScreenWrapper = styled.div<{ enabled?: boolean }>`
-  ${({ enabled }) =>
+const FullScreenWrapper = styled.div<{ enabled?: boolean; animationType?: DialogAnimationType }>`
+  ${({ enabled, animationType }) =>
     enabled &&
     css`
       align-items: center;
+      ${animationType === DialogAnimationType.FADE ? fadeAnimationCss : ''}
```

Comment:
`FullScreenWrapper` is the backdrop for full screen modals - this css is only applied if we enable the backdrop via the prop